### ISSUE TITLE
Add a yield block

### ISSUE THREAD

**Issue by NoMod-Programming** [GRA0007/Blue#8]
Now that we have custom c blocks, I think that we should add a yield block, that way custom c blocks that loop could be fully functional and work like normal scratch c-blocks. The problem is, some users may not understand what `yield` means, and won't know how to use it. So should it be added?

**Comment by GRA0007** [comment]
I think the point of this scratch mod is to allow the users to do much more than they ever could in vanilla scratch, so yes, I think it would be a good idea to add it. If someone using the mod doesn't understand what it does, then they just won't use it. I like to think of Blue as scratch for more 'advanced' scratchers, and hopefully when modshare is up and running again and Blue is out of beta, these advanced scratchers will be able to create amazing projects and share them for anyone to see.

tl;dr Yes add them ;)

**Comment by NoMod-Programming** [comment]
Done.
